source/includes/_tipser-elements-analytics.md

```diff
@@ -1,189 +1,308 @@
-(first clicking on a product title and then seeing the details).
-
-### GA event format ###
-
-Category         | Action      | Standard Enhanced Ecommerce event?
----------        | ----------- | -----------
-`E-commerce`     | `click`     | Yes (with one product attached)
-`E-commerce`     | `detail`    | Yes (with one product attached)
-
-## Add to cart
-
-This event is emitted when product is added to cart on a product dialog or checkout dialog.
-
-[![](add_to_cart_event.png)](/images/add_to_cart_event.png)
-
-_In the screenshot above the user has clicked "add to cart" button which resulted in emitting "Add to cart" event._
-
-### GA event format ###
-
-Category         | Action         | Standard Enhanced Ecommerce event?
----------        | -----------    | -----------        
-`E-commerce`     | `add`          | Yes (with one product attached) 
-
-## Remove from cart
-
-This event is emitted when product is removed from cart in checkout dialog.
-
-### GA event format ###
-
-Category         | Action         | Standard Enhanced Ecommerce event?
----------        | -----------    | -----------       
-`E-commerce`     | `remove`       | Yes (with one product attached)
-
-## Checkout
-
-This event is emitted when checkout dialog is opened with a product (or several products).
-
-[![](product_checkout.png)](/images/product_checkout.png)
-
-_In the screenshot above checkout dialog has been displayed and "Checkout" event has been emitted for a product "Retexturising SERUM"_
-
-### Additonal notes ###
-
-Please note that "checkout" is not the same thing as "purchase". The event is emitted even if the product checkout is never finalized.
-The event is emitted every time checkout dialog is opened. This means that there may be several "Checkout" events emitted for product that was purchased just 
-once or that was never purchased.
-
-### GA event format ###
-
-Category         | Action         | Standard Enhanced Ecommerce event?     
----------        | -----------    | -----------               
-`E-commerce`     | `checkout`     | Yes (with one or many products attached)
-
-
-## Purchase
-
-This event is emitted when a product (or several products) is purchased.
-
-[![](purchase_event.png)](/images/purchase_event.png)
-
-_In the screenshot above the user has finalized the payment and the order confirmation dialog has been displayed. "Purchase" event has been emitted._
-
-### GA event format ###
-
-Category         | Action         | Standard Enhanced Ecommerce event?        
----------        | -----------    | -----------               
-`E-commerce`     | `checkout`     | Yes (with one or many products attached)
+# Tipser Elements Analytics
+
+Tipser has implemented an event for the Tipser Elements Analytics that users of the script can listen to in order to gather analytics about Tipser interactions and feed them into their own analytics engines.
```

Comment:
Maybe: Tipser (Elements) provides an analytics event that users of the script...?

Reply:
Actually users of the script is... Tipser Elements. NPM package is Tipser React Elements. But it actually works for ppl who are integrated with Tipser SDK.

Okay I will change it to, is this okay:?
```
Tipser has implemented an event for the Tipser Elements. Its users can listen to it in order to gather analytics about Tipser interactions and feed them into their own analytics engines.

> Tipser React Elements and Tipser SDK aslo also come with built in Analytics. Tipser SDK, due to its nature, supports only subset of interactions.
```

Reply:
Maybe: an event for the Tipser Elements -> an *analytics* event for the Tipser Elements, to give more idea what kind of event it is? But it's a detail that I won't argue about. Overall it looks good.

Reply:
Please suggest the description as whole (the whole paragraph). I think adding `analytics` word in the first sentence make sense, but then it will duplicate the word in the second sentence and that would sound not nice. I mean by changing first sentence, we need to consider changing the whole paragraph, and I think we would need after your suggestion.